src/app/modules/map-config/services/map-layer-form-builder/map-layer-form-builder.service.ts

```diff
@@ -396,11 +396,11 @@ export class MapLayerAllTypesFormGroup extends ConfigFormGroup {
     super({
       collectionStep: new ConfigFormGroup({
         collection: new SelectFormControl(
-          collections.length === 1 ? collections[0] : '',
+          collection,
           marker('Collection'),
           '',
           false,
-          collections.map(c => ({ label: c, value: c }))
+          [].map(c => ({ label: c, value: c }))
```

Comment:
It seems this do nothing

Reply:
yes, for now this step is hidden

I'll change it in a next PR